### PR TITLE
updated docs re: gov a11y review

### PR DESCRIPTION
**Pull request by ADPennington**
### Summary of Changes
_Updated docs related to gov a11y review_
### Updated documents
- [x] `git workflow ADR` now includes suggestions for capturing documentation for Raft's a11y review, indicators of frontend epic completeness, and government's review process flow
- [x] `our workflow` now recognizes that COR review will be conducted on PRs on as-needed basis
- [x] `how-government-will-test-a11y` acknowledges updated decision on when gov a11y review will be conducted and documentation to be considered. 
- [x] `qasp checklist` includes new questions in accessibility section to track completeness of frontend epic and Raft a11y documentation trail

### CODE REVIEW

File: docs/Technical-Documentation/qasp-operational-checklist.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ As Product Owner, @lfrohlich will decide if ACs are met.
 > *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*
 
 > *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*
-
++ [ ] Does this PR complete the epic? 
++ [ ] Are links included to any other gov-approved PRs associated with epic?
 + [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
```
Comment (ADPennington) [inline docs/Technical-Documentation/qasp-operational-checklist.md:42]:
```suggestion
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
```

**Comment by ADPennington** [comment]
@lfrohlich and @alexsoble in addition to your review of these changes, please also feel free to suggest additional documents that might be worth updating for this. I also think it could be helpful to suggest tagging @ttran-hub and @jorgegonzalez in advance when Raft plans to hop into wednesday a11y office hours, but these are informal and something that could be accomplished in Teams.

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ A contributor to the TDP project would always use the following steps to propose
     * Add in-line comment to the file changes to provide context for the proposed changes
     * Ensure there are no merge conflicts 
     * Ensure CI/CD pipelines are green
+    * For frontend PRs, ensure that documentation trail from Raft's a11y review associated with epic is included and indicate if PR completes this epic.
```
Comment (alexsoble) [inline docs/Architecture Decision Record/009-git-workflow.md:37]: 👍 👍

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 3]
```diff
@@ -49,8 +50,9 @@ A contributor to the TDP project would always use the following steps to propose
     * removes `raft-review` label
     * adds `qasp-review` label
     * assigns the Government as the reviewer:
-        * For backend development work assign `alexsoble`, `adpennington` and `lfrohlich`
-        * For frontend work, assign the same as backend work but also include `iamjolly` and `ttran-hub` for a11y review
+        * For backend and frontend development work assign `adpennington` and `alexsoble`
```
Comment (alexsoble) [inline docs/Architecture Decision Record/009-git-workflow.md:53]: Not related but just noting we'll want to remove me from this line during offboarding ... need to start building up an offboarding checklist! 😢

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 4]
```diff
@@ -49,8 +50,9 @@ A contributor to the TDP project would always use the following steps to propose
...
+        * For frontend work submitted for review but does not yet complete the epic, `adpennington` will sign off on Raft's a11y documentation (e.g. screen captures with VoiceOver utility, summary of review, etc.) and cc: `ttran-hub`
+        * For frontend work submitted for review that completes the epic, `adpennington` will complete code review, tag `ttran-hub` + `iamjolly` via comment, and add `a11y` label to PR
```
Comment (alexsoble) [inline docs/Architecture Decision Record/009-git-workflow.md:55]: One suggestion, would it make sense to mention here that Thomas and Robert will do manual review using Accessibility Insights for work that represents a completed Epic?

File: docs/Technical-Documentation/how-government-will-test-a11y.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ We will check new pages, features, and interactions added during the course of e
   - VoiceOver on Safari, Mac OS
   - VoiceOver on iOS
 
-We won't re-rest pages or elements that have previously been tested and haven't changed since, such as header or footer elements.
+We will consider all documentation from Raft's a11y reviews included in each PR associated with the epic. We won't re-test pages or elements that have previously been tested and haven't changed since, such as header or footer elements.
```
Comment (alexsoble) [inline docs/Technical-Documentation/how-government-will-test-a11y.md:27]: Good typo fix, thanks for catching that! 😂

**Review by alexsoble — APPROVED** [review]
Left a few inline comments, overall LGTM!

File: docs/Architecture Decision Record/009-git-workflow.md (hunk 4 above)
Comment (alexsoble) [reply docs/Architecture Decision Record/009-git-workflow.md:55]: Maybe not since this is just about the git flow

**Comment by lfrohlich** [comment]
> @lfrohlich and @alexsoble in addition to your review of these changes, please also feel free to suggest additional documents that might be worth updating for this. I also think it could be helpful to suggest tagging @ttran-hub and @jorgegonzalez in advance when Raft plans to hop into wednesday a11y office hours, but these are informal and something that could be accomplished in Teams.

👍  should @jorgegonzalez  be @iamjolly here ?

File: docs/Architecture Decision Record/009-git-workflow.md (hunk 4 above)
Comment (lfrohlich) [reply docs/Architecture Decision Record/009-git-workflow.md:55]: we could refer to this: https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/how-government-will-test-a11y.md

**Comment by ADPennington** [comment]
> > @lfrohlich and @alexsoble in addition to your review of these changes, please also feel free to suggest additional documents that might be worth updating for this. I also think it could be helpful to suggest tagging @ttran-hub and @jorgegonzalez in advance when Raft plans to hop into wednesday a11y office hours, but these are informal and something that could be accomplished in Teams.
> 
> 👍 should @jorgegonzalez be @iamjolly here ?

I meant Jorge 😄since he usually ensures feature work is deployed in advance of office hours. but in theory whomever the lead developer is could be tagged.

File: docs/Technical-Documentation/how-government-will-test-a11y.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ Our web testing tool will be [Accessibility Insights](https://accessibilityinsig
 
 Accessibility Insights includes both a [Fast Pass](https://accessibilityinsights.io/docs/en/web/getstarted/fastpass/) tool and a comprehensive [Assessment](https://accessibilityinsights.io/docs/en/web/getstarted/assessment/) tool.
 
-We will check new pages, features, and interactions added during the course of each sprint using:
+We will conduct accessibility review at the epic-level (i.e., when the last  or only feature associated with the epic has been submitted for QASP review). We will check new pages, features, and interactions added during the course of each sprint using:
```
Comment (lfrohlich) [inline docs/Technical-Documentation/how-government-will-test-a11y.md:18]: Does this mean we'll be using accessibility insights each sprint still?

**Review by lfrohlich — APPROVED** [review]
Looks great! added a few minor comments

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # 9. Git Workflow
 
-Date: 2021-02-23
+Date: 2021-04-22
```
Comment (lfrohlich) [inline docs/Architecture Decision Record/009-git-workflow.md:3]: Should we keep the original date and say "updated 4-22" ?

Comment (ADPennington) [inline docs/Architecture Decision Record/009-git-workflow.md:3]:
```suggestion
Date: 2021-02-23 (_updated 2021-04-22_)
```

File: docs/Architecture Decision Record/009-git-workflow.md (hunk 4 above)
Comment (ADPennington) [inline docs/Architecture Decision Record/009-git-workflow.md:55]:
```suggestion
        * For frontend work submitted for review that completes the epic, `adpennington` will complete code review, tag `ttran-hub` + `iamjolly` via comment, and add `a11y` label to PR. Gov a11y review team will use accessibility insights for manual testing. More information about this process is described [here](https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/how-government-will-test-a11y.md) 
```

File: docs/Architecture Decision Record/009-git-workflow.md [hunk 8]
```diff
@@ -49,8 +50,9 @@ A contributor to the TDP project would always use the following steps to propose
     * removes `raft-review` label
     * adds `qasp-review` label
     * assigns the Government as the reviewer:
-        * For backend development work assign `alexsoble`, `adpennington` and `lfrohlich`
-        * For frontend work, assign the same as backend work but also include `iamjolly` and `ttran-hub` for a11y review
+        * For backend and frontend development work assign `adpennington` and `alexsoble`
+        * For frontend work submitted for review but does not yet complete the epic, `adpennington` will sign off on Raft's a11y documentation (e.g. screen captures with VoiceOver utility, summary of review, etc.) and cc: `ttran-hub`
+        * For frontend work submitted for review that completes the epic, `adpennington` will complete code review, tag `ttran-hub` + `iamjolly` via comment, and add `a11y` label to PR. Gov a11y review team will use accessibility insights for manual testing. More information about this process is described [here](https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/how-government-will-test-a11y.md)
```
Comment (ADPennington) [inline docs/Architecture Decision Record/009-git-workflow.md:55]:
```suggestion
        * For frontend work submitted for review that completes the epic, `adpennington` will complete code review, tag `ttran-hub` + `iamjolly` via comment, and add `a11y` label to PR. Gov a11y review team will use accessibility insights for manual testing. More information about how and when this process will be carried out is described [here.](https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/how-government-will-test-a11y.md) 
```

**Review by carltonsmith — APPROVED** [review]
LGTM!

**Pull request merged**